README.md

```diff
@@ -24,7 +24,7 @@ Run the app again and click "Restore Default Status Bar". Resetting the iOS Simu
 Yes! SimulatorStatusMagic is also available through [CocoaPods](http://cocoapods.org). To install, simply add the following line to your Podfile:
 
 ```ruby
-pod 'SimulatorStatusMagic', :configurations => ['Debug']
+pod 'SimulatorStatusMagic', configurations: ['Debug']
```

Comment:
In my opinion, I'd suggest to keep it consistent with all the guides on CocoaPods using the hash rocket. People who don't know Ruby may get confused with the differences, especially when trying to merge two different styles together while looking at the guides.

Example:

```ruby
pod 'SimulatorStatusMagic', configurations: ['Debug'], :git => 'https://github.com/shinydevelopment/SimulatorStatusMagic'
```